notary_now_api/migrations/0003_auto_20200215_2109.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.0.3 on 2020-02-15 21:09
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
✅ Add migrations for the database
Interesting how Django only needs the name `user` and not `user_id`